.swiftlint.yml

```diff
@@ -0,0 +1,10 @@
+disabled_rules:
+  - trailing_whitespace
+excluded:
+  - Carthage
+  - fr-swiftTests
+  - fr-swiftUITests
```

Comment:
This has the old name !
Use `WBooksTests` and `WBooksUITests`

Reply:
I will add this change in the next PR